README.md

```diff
@@ -560,7 +560,16 @@ In order to use the same instance of a class concurrently in multiple threads th
 
 The level of thread safety within Exiv2 varies depending on the type of metadata: The Exif and IPTC code is reentrant. The XMP code uses the Adobe XMP toolkit (XMP SDK), which according to its documentation is thread-safe. It actually uses mutexes to serialize critical sections. However, the XMP SDK initialisation function is not mutex protected, thus Exiv2::XmpParser::initialize is not thread-safe. In addition, Exiv2::XmpProperties::registerNs writes to a static class variable, and is also not thread-safe.
 
-Therefore, multi-threaded applications need to ensure that these two XMP functions are serialized, e.g., by calling them from an initialization section which is run before any threads are started.
+Therefore, multi-threaded applications need to ensure that these two XMP functions are serialized, e.g., by calling them from an initialization section which is run before any threads are started.  All exiv2 sample applications begin with:
+
+```c++
```

Comment:
I think this is not supported and instead of using `c++` we should use `cpp`

Reply:
Right.  I'll fix that.  Thanks.

I've discovered something totally new #1219 which I'm investigating.